src/Cmgmyr/Messenger/Models/Message.php

```diff
@@ -94,4 +94,25 @@ public function recipients()
     {
         return $this->participants()->where('user_id', '!=', $this->user_id);
     }
+
+    /**
+     * Prepares Builder for unread user messages
+     *
+     * @param $query
+     * @param $userId
+     * @return mixed
+     */
+    public function scopeUnreadIncomingForUser($query, $userId)
```

Comment:
Maybe it will be better to name it `scopeUnreadMessagesForUser`?

Reply:
@a-komarev you are probably right - `scopeUnreadIncomingForUser` is too long, but it represents the scope better as for me as we are excluding outbound messages.

Reply:
`scopeUnreadMessagesForUser` works for me.